packages/angular-cli/tasks/server/universal-dev-server.ts

```diff
@@ -0,0 +1,64 @@
+import * as path from 'path';
+const nodemon = require('nodemon');
+const tinyLr = require('tiny-lr');
+
+export class UniversalDevServer {
+  private appPath: string = '';
+  private nodemonConfig: any;
+  private demon: any;
+  private liveServer: any;
+
+  constructor(private webpackCompiler: any, private webpackDevServerConfiguration: any) {
```

Comment:
What is the purpose for not using webpack-dev-server. Additionally, if you are wanting to use an express server there is webpack-dev-middleware as well which has a lot of the api configuration to hook webpack into your existing express server.